source/features/highlight-collaborators-and-own-conversations.css

```diff
@@ -1,5 +1,11 @@
 .rgh-collaborator {
+	display: inline-block;
 	border: 1px solid var(--color-border-default);
 	border-radius: 2em;
-	padding: 2px 5px;
+	padding: 0 5px;
+}
+
+/* Extra padding for `small-user-avatars` feature */
+.rgh-collaborator:has(.avatar) {
+	padding-bottom: 2px;
```

Comment:
I don't think any of this CSS should be used, it's causing the misalignment and layout shift. It's best to align the image directly

Reply:
Here's a zero-shift image that can be inlined there (rough values, just show it's possible)


```html
<img class="from-avatar avatar-user" src="https://avatars.githubusercontent.com/u/44045911?s=40&amp;v=4" width="14" height="14" alt="@kidonng" style="
    margin-right: 3px;
    vertical-align: -2px;
    margin-left: -2.6px;
">
```

![fixed](https://user-images.githubusercontent.com/1402241/230943745-757c84e7-22ec-41a8-92a6-c0ebab58435e.gif)

Tested in Chrome only, may need `inline-flex` or `em` to avoid font weirdness

Reply:
I really want to stick to 16px as that's what we use for `reaction-avatars`, but yeah 14px is fine, this place is just too small.

This is how it looks like now, without any magic numbers.

<img width="643" alt="image" src="https://user-images.githubusercontent.com/44045911/230949759-7d423a14-ab8e-44b0-b882-3f8122a50b78.png">

Reply:
Let's not bring these PRs into loops. I already commented on that, it's not aligned and it should be fixed

Reply:
This is the best I can do:

<img width="629" alt="image" src="https://user-images.githubusercontent.com/44045911/230956600-32bd3dd0-4ec0-4393-acd4-e4fbb2d0cd12.png">

Perhaps I'm really bad at CSS, I just don't know how to

- Not change `highlight-collaborators-and-own-conversations`
- Not causing content shifts
- And have all these places aligned

<img width="405" alt="image" src="https://user-images.githubusercontent.com/44045911/230957871-71f4c00d-dae7-4f25-9c08-2aa24d6c8608.png">

So I don't get which places you want it aligned.

Reply:
I mean I want it as it appears in the gif I provided and the magic-numbered CSS. It's ok, I can look into a proper solution later if there's no easy way to achieve it.  Dealing with text alignment is always a pain in CSS